WordPressVIPMinimum/Sniffs/Security/ProperEscapingFunctionSniff.php

```diff
@@ -53,8 +54,27 @@ public function process_token( $stackPtr ) {
 
 		$function_name = $this->tokens[ $stackPtr ]['content'];
 
-		$echo_or_string_concat = $this->phpcsFile->findPrevious( Tokens::$emptyTokens, $stackPtr - 1, null, true );
+		$data = [ $function_name ];
 
+		if ( $function_name === 'esc_attr' || $function_name === 'esc_attr__' ) {
+			// Find esc_attr being used between HTML tags.
+			$tokens = array_merge(
```

Comment:
Why are you trying to _sort of_ do the same thing were as the code block below is already doing ?

Is there a specific reason why this check has to be done ahead of the rest of the checks ?